test/DynamoCoreTests/Performance/PerformanceTests.cs

```diff
@@ -135,5 +319,121 @@ private void CheckForDummyNodes(WorkspaceModel ws)
                 Assert.Inconclusive("The Workspace contains dummy nodes for: " + string.Join(",", dummyNodes.Select(n => n.Name).ToArray()));
             }
         }
+
+        private string FormatValue(long value)
+        {
+            if (value > 1000000)
+            {
+                return string.Format("{0:0.00}M", value / 1000000.0);
+            }
+            if (value > 1000)
+            {
+                return string.Format("{0:0.00}k", value / 1000.0);
+            }
+
+            return string.Format("{0:0.00}", value);
+        }
+
+        private void LotsOfColoredStuffSimplified()
+        {
+            var seeds = Enumerable.Range(0, 26).Select(x => x * 2).ToList();
+
+            var points = seeds.Select(x => seeds.Select(y => seeds.Select(z => Point.ByCoordinates(x, y, z)).ToList()).ToList()).ToList();
+
```

Comment:
Why all the ToLists() calls ?

Reply:
@pinzart90 My reasoning when I did this was that I wanted to see what I was seeing in Dynamo after each node. I thought that was being a fair comparison. 

But we might want to change this in light of what we discussed last week. I could change this to only provide Lists for what we would consider to be the output nodes? What do you think?

Reply:
I guess there is value in keeping the operations that we know are necessary or that definitely cannot be optimized away.
It is tough to say when it is `fair` enough.
But for the purposes of comparing the ideal compiled state (what you hardcoded) vs the what we could potentially do with compiling to direct function calls I think it is `fair` enough.

I say it is good enough with the ToList() calls